smartsim/log.py

```diff
@@ -205,7 +204,7 @@ def get_logger(
     """
     # if name is None, then logger is the root logger
     # if not root logger, get the name of file without prefix.
-    user_log_level = _get_log_level()
+    user_log_level = os.environ.get("SMARTSIM_LOG_LEVEL", "info")
```

Comment:
I wonder what you & @MattToast think about trying to move all `os.environ['xyz']` calls into the `smartsim._core.config.Config`

Reply:
+1

Reply:
It was already there...